recipes/glfw/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration, ConanException
+
+
+class GlfwConan(ConanFile):
+    name = "glfw"
+    description = "GLFW is a free, Open Source, multi-platform library for OpenGL, OpenGL ES and Vulkan" \
+                  "application development. It provides a simple, platform-independent API for creating" \
+                  "windows, contexts and surfaces, reading input, handling events, etc."
+    settings = "os", "arch", "build_type", "compiler"
+    license = "Zlib"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/glfw/glfw"
+    topics = ("conan", "gflw", "opengl", "vulkan", "opengl-es")
+    exports = "LICENSE"
+    generators = "cmake"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    _source_subfolder = "source_subfolder"
+    _build_subfolder = "build_subfolder"
+
+    def _check_system_libs(self, libs, install_command):
+        installer = tools.SystemPackageTool()
+        missing_system_libs=[]
+        for package_name in libs:
+            if not installer.installed(package_name):
+                missing_system_libs.append(package_name)
+        if len(missing_system_libs) > 0:
+            raise ConanException(
+                'System libraries missing: {0} not found. You can install using: "sudo {1} {0}"'
+                .format(" ".join(missing_system_libs), install_command))
+
+    def system_requirements(self):
+        if self.settings.os == "Linux":
+            package_tool = tools.SystemPackageTool(conanfile=self, default_mode="verify")
+            libs_name = ""
+            os_info = tools.OSInfo()
+            if os_info.with_apt:
+                libs_name = "libx11-dev libxrandr-dev libxinerama-dev libxkbcommon-dev libxcursor-dev " \
```

Comment:
Is the plan to have a virtual package for x11 abandoned ?

Reply:
For the moment, I think we will be trying this approach, setting `default_mode="verify"` in `SystemPackageTool`  so we don't force users to install the system requires until we have a more clear idea on what to do with this kind of packages.